src/portal_visualization/builders/anndata_builders.py

```diff
@@ -74,17 +76,35 @@ def get_conf_cells(self):
             gene_alias=gene_alias
         ))
 
-        vc = self._setup_anndata_view_config(vc, dataset)
+        vc = self._setup_anndata_view_config(vc, dataset, marker_gene)
         return get_conf_cells(vc)
 
-    def _setup_anndata_view_config(self, vc, dataset):
-        vc.add_view(cm.SCATTERPLOT, dataset=dataset, mapping="UMAP", x=0, y=0, w=4, h=6)
-        vc.add_view(cm.CELL_SET_EXPRESSION, dataset=dataset, x=4, y=0, w=5, h=6)
-        vc.add_view(cm.CELL_SETS, dataset=dataset, x=9, y=0, w=3, h=3)
-        vc.add_view(cm.GENES, dataset=dataset, x=9, y=4, w=3, h=3)
-        vc.add_view(cm.HEATMAP, dataset=dataset, x=0, y=6, w=12, h=4)
+    def _setup_anndata_view_config(self, vc, dataset, marker_gene=None):
+        scatterplot = vc.add_view(
+            cm.SCATTERPLOT, dataset=dataset, mapping="UMAP", x=0, y=0, w=self._scatterplot_w, h=6)
+        cell_sets = vc.add_view(
+            cm.CELL_SETS, dataset=dataset, x=9, y=0, w=3, h=3)
+        gene_list = vc.add_view(
+            cm.GENES, dataset=dataset, x=9, y=4, w=3, h=3)
+        cell_sets_expr = vc.add_view(
+            cm.CELL_SET_EXPRESSION, dataset=dataset, x=7, y=6, w=5, h=4)
+
+        vc.add_view(cm.HEATMAP, dataset=dataset, x=0, y=6, w=7, h=4)
```

Comment:
Why is `HEATMAP` handled differently than the other views?

Reply:
I was going to say "because there is no need to coordinate it with the others based on gene"... but there's no reason we couldn't highlight a row or column for the gene. I'll ask on slack.

Reply:
- Filed https://github.com/vitessce/vitessce/issues/1298